relay-statsd/src/types.rs

```diff
@@ -0,0 +1,221 @@
+/// A metric for capturing timings.
```

Comment:
Symbolicator does not have enum types to declare metrics, so I copied the contents of this file from https://github.com/getsentry/relay/blob/aa7d644263cd6f3249ebae85c836b0754d3f4da5/relay-statsd/src/lib.rs#L300-L530.